package/src/macos/installer.ts

```diff
@@ -78,7 +78,7 @@ export async function makeInstallerMac(config: Configuration) {
     // Sign these executable / binary files
     // and include our entitlements declaration
     const signWithEntitlements = [
-      join(config.directoryInfo.bin, "tools", "deno"),
+      Deno.env.get("QUARTO_DENO_PATH"),
```

Comment:
probably don't need to sign the conda binary. This file is probably fine to revert.